src/python/tests/xrefs/test_xref_source.py

```diff
@@ -0,0 +1,41 @@
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Unit testing of `ensembl.xrefs.xref_source_db_model` module."""
+
+
+import pytest
+
+from ensembl.xrefs.xref_source_db_model import Base
+from ensembl.utils.database import UnitTestDB
+
+
+@pytest.mark.parametrize("test_dbs", [[{"src": "core_db"}]], indirect=True)
+class TestCoreModels:
+    """Tests the `~ensembl.xrefs.xref_source_db_model` ORMs"""
```

Comment:
```suggestion
    """Tests the `ensembl.xrefs.xref_source_db_model` ORMs"""
```
Unfortunately that syntax is not supported by mkdocs.